custom_platforms.js

```diff
@@ -223,7 +248,11 @@ module.exports = {
                 "\\$protobuf",
                 "important,protobuf",
                 "\\$extension",
-                ",extension"
+                ",extension",
```

Comment:
It seems that `",domain=\/",` was accidentally removed from exclusions for Opera extension - https://github.com/AdguardTeam/FiltersRegistry/pull/806/commits/710e969aa8d916708d35398af27f675c83bcb4a9#diff-6d9b3e53da2e741716b096e56da5b27d6fb4daefcfce8e03d5a2f565a447da0fL253
```suggestion
                ",extension",
                ",domain=\/",
```

Reply:
But this is Chromium based Opera. It was accidentally _added_ :)